eng/common/pipelines/templates/jobs/sdk-generation.yml

```diff
@@ -0,0 +1,160 @@
+parameters:
+  - name: SpecRepoBranch
+    type: string
+    default: 'main'
+  - name: SpecRepoCommit
+    type: string
+    default: 'HEAD'
+  - name: SdkRepoOwner
```

Comment:
Could you use `SdkRepoUrl` as the input parameter then parse the value to get the owner and the name instead?

Reply:
We could do this, but it seems unnecessary since it involves adding another pipeline step to parse the url and create the require pipeline variables. I think its simpler and neater to just parse the repo information parts as individual parameters plus we don't have to run an extra script step.

Reply:
We will need a script to parse the repository Uri when the pipeline is triggered by a spec pull request. In addition, combining these two parameters to a single one simplifies the user interface.